hadoop-hdds/docs/content/feature/Quota.md

```diff
@@ -32,11 +32,16 @@ So far, we know that Ozone allows users to create volumes, buckets, and keys. A
 1. Storage Space level quota
```

Comment:
The namespace quota introduction can also be added. We could file a new JIRA for tacking this.

Reply:
I can add namespace quota to this documentation: https://issues.apache.org/jira/browse/HDDS-4594. Before adding to documentation, I will first finish namespace support on bucket: https://issues.apache.org/jira/browse/HDDS-4277